scripts/make_inventory_file

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
```

Comment:
Can we rename this file to scripts/tripleo-inventory to keep container-validations project-agnostic?
In the future the scripts directory could have other scripts like scripts/openshift-node-inventory or similar.

Reply:
Fixed.